src/webui/frontend/package.json

```diff
@@ -51,7 +51,7 @@
   },
   "proxy": "http://localhost:8000",
   "devDependencies": {
-    "@types/classnames": "^2.2.10",
+    "@types/classnames": "2.3.1",
```

Comment:
did you mean to remove the `^` here?

Reply:
Actually no, will push with a `^`

Reply:
Actually no, will push a change for this

Reply:
Actually no, will push a change for this, but the package has been deprecated after this.